playground/AzureStorageEndToEnd/AzureStorageEndToEnd.AppHost/Program.cs

```diff
@@ -9,11 +9,14 @@
 });
 
 var blobs = storage.AddBlobs("blobs");
+var blobContainer = blobs.AddBlobContainer("mycontainer");
+
 var queues = storage.AddQueues("queues");
 
 builder.AddProject<Projects.AzureStorageEndToEnd_ApiService>("api")
        .WithExternalHttpEndpoints()
        .WithReference(blobs).WaitFor(blobs)
+       .WithReference(blobContainer).WaitFor(blobContainer)
```

Comment:
This isn't needed, yet, right? We aren't referencing/using the connection string for `mycontainer` in the playground app.